admin/server.js

```diff
@@ -67,11 +67,9 @@ function onError (error) {
     case 'EACCES':
       console.error(bind + ' requires elevated privileges')
       process.exit(1)
-      break
     case 'EADDRINUSE':
       console.error(bind + ' is already in use')
       process.exit(1)
-      break
```

Comment:
Isn't the linter complaining about this being removed?

Reply:
No.  Checked with standard v14.3.1, and v12.0.1